### DISCUSSION TITLE
SPAMMING

### DISCUSSION THREAD

**Discussion by scotty2541** [emqx/emqx#10176] [General]
Why is this company, EMQX, decided it okay to START SPAMAMING PEOPLE??   
People who DID NOT SIGN UP? Nor doing ANYTHING related to the INTRUSIVE SPAM they are PUSHING??

And then have NO contact information on their web site?  Leaving NO WAY to give them feedback except to try using other channels?

**Comment by gujun720** [comment]
Hi Scott (@scotty2541) ,

This is Jun. I am responsible for the community relations of EMQ projects.
I am so sorry for disturbing you with the email. I will definitely investigate why this happened to you and remove you from the mail list.

If you still get those emails by mistake, please forward it to me (jun.gu@emqx.io).

Have a nice day :)

**Comment by scotty2541** [comment]
I apologize for using this board, but your web site doesn't have any contact info.  So I am so frustrated getting pointless garbage, I resorted to social media and other ways to complain.

But I found something on the ".com" version of the site. And I got a message from someone named "Ben". 

Places like MailChimp, Constant Contact, Royal Media, MkToMail, CloudFare,  ...  appear to be SELLING my info to spammers....   adding my name to charge more for bigger junk mail lists...  I get SPAM on topics that are LIGHT YEARS away from my role.  Very annoying.

As I told Ben, to the best of my knowledge, I never registered, signed up, subscribed... (if I did,  please remind me) I have not dealt with this subject matter in many years.  I'd like to know who is selling my info.   No one else has offered to try to trace how this is happening.

**Reply by gujun720** [reply]
I appreciate you bringing this incident to us. We are trying to make money to ensure the sustainability of our open-source projects (we have developed the emqx project for more than a decade).

There is no way we could do business by randomly sending emails to strangers. We will carefully review where we get your contact. 

If those agents gave us the contact as you mention, then it's no good for both of us. You got spam and we got irrelevant information.☹️
